### PR TITLE
added Sign up and Log in pages

### PR DESCRIPTION
**Pull request by DivyanshSareen**
added `LoginForm` and `SignupForm` components under `LoginPage` and `SignupPage`.

Note: Please ignore the merge conflict

Signup Page: https://deploy-preview-3--courageous-sundae-3661d6.netlify.app/

Login Page:
image

### CODE REVIEW

File: src/components/LoginForm.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Nameimport "../styles/login.css";
+
+export default function LoginForm(){
```
Comment (amartya-rs) [inline src/components/LoginForm.jsx]: I would suggest you to use arrow functions.

File: src/pages/LoginPage.jsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import Nav from "../components/Nav";
+import LoginForm from "../components/LoginForm";
+
+export default function LoginPage(){
+    return (<>
+        <Nav/>
```
Comment (amartya-rs) [inline src/pages/LoginPage.jsx]: use a good code formatter like Prettier it makes your code more presentable. For eg: 

prettier
.

File: src/components/SignupForm.jsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import "../styles/login.css";
+
+export default function SignupForm(){
+    return(
```
Comment (amartya-rs) [inline src/components/SignupForm.jsx]: Same here, use arrow functions.

**Review by amartya-rs — APPROVED** [review]
Good to merge with the suggested changes.

**Review by tonymike99 — APPROVED** [review]
The UI looks great!
Look into the suggestions that the others gave. And you are good to go.

File: src/components/LoginForm.jsx (hunk 1 above)
Comment (DivyanshSareen) [reply src/components/LoginForm.jsx]: will use them in a later commit 👍

**Pull request merged**